modules/tracker/models/base/ScalarModelBase.php

```diff
@@ -32,6 +32,7 @@ public function __construct()
             'scalar_id' => array('type' => MIDAS_DATA),
             'trend_id' => array('type' => MIDAS_DATA),
             'user_id' => array('type' => MIDAS_DATA),
```

Comment:
Do you mean the `user_id` here or everything about the model layer? :smiling_imp: